### PR TITLE
chore: Adopt inclusive language in sanitize_redirect.js

### PR DESCRIPTION
**Pull request by gkartalis**
# Description 

Adopts inclusive language in `sanitize_redirect.js`, resolves issue: https://github.com/artsy/artsy-passport/issues/188 as per https://github.com/artsy/README/issues/427

### CODE REVIEW

**Review by pvinis — APPROVED** [review]

**Review by sonecabr — APPROVED** [review]
LGTM

File: lib/app/sanitize_redirect.js [hunk 1]
```diff
@@ -6,8 +6,8 @@ const _ = require('underscore')
 const parse = require('url').parse
 
 const redirectFallback = '/'
-const whitelistHosts = ['internal', 'localhost', 'artsy.net']
-const whitelistProtocols = ['http:', 'https:', null]
+const allowlistHosts = ['internal', 'localhost', 'artsy.net']
```
Comment (araujobarret) [inline lib/app/sanitize_redirect.js:9]: Nit: allowedListHosts, allowedListProtocols

Comment (pvinis) [reply lib/app/sanitize_redirect.js:9]: I think it's ok as it is, but I guess it's just opinions. same way as whitelist was not whiteList, I think allowlist can be a word.

Comment (araujobarret) [reply lib/app/sanitize_redirect.js:9]: Sure, I think I saw this word before and I'm okay with it :)

Comment (gkartalis) [reply lib/app/sanitize_redirect.js:9]: 🤩

**Comment by artsyit** [comment]
:rocket: PR was released in [`v3.5.3`](https://github.com/artsy/artsy-passport/releases/tag/v3.5.3) :rocket:

**Pull request merged**